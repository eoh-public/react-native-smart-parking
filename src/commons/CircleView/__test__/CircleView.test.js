import React from 'react';
import renderer, { act } from 'react-test-renderer';
import { CircleView } from '../index';

describe('Test circle view', () => {
  let tree;
  test('create circle view', () => {
    act(() => {
      tree = renderer.create(<CircleView size={5} />);
    });
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
