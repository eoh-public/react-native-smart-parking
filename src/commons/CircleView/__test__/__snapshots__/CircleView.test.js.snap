// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test circle view create circle view 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderColor": "white",
        "borderRadius": 2.5,
        "borderWidth": 0,
        "height": 5,
        "width": 5,
      },
      undefined,
      undefined,
    ]
  }
/>
`;
